aiida_vasp/calcs/vasp.py

```diff
@@ -5,7 +5,7 @@
 The calculation class that prepares a specific VASP calculation.
 """
 #encoding: utf-8
-# pylint: disable=abstract-method
+# pylint: disable=abstract-method, super-with-arguments
```

Comment:
New version of pylint?

Reply:
Yes, sorry about that, have been trying to follow the convention that aiida-quantumespresso uses, and I think they have 2.6

Reply:
For aiida-vasp we try to stick to the aiida-core.

Reply:
I think `pip install -e .[pre-commit]` should give you that environment. Also with the `tests` extra for the tests.